snapcraft.yaml

```diff
@@ -23,11 +20,16 @@ parts:
       - voluptuous
     prime:
       - bin/ubuntu-image
+      - bin/dd
+      - bin/cp
```

Comment:
Ok, yeah, this was part of my first try to actually include 'all' in the snap. Will change that.